lyse/__init__.py

```diff
@@ -388,11 +388,15 @@ def get_attrs(self, group):
                 raise Exception('The group \'%s\' does not exist'%group)
             return get_attributes(h5_file[group])
 
-    def get_trace(self,name):
+    def get_trace(self, name, raw_data=False):
         """Return the saved data trace `name`.
         
         Args:
             name (str): Name of saved data trace to get.
+            
+        Kwargs:
+            raw_data (bool): option to return the h5_data directly
+            without interperting it as a 2-D time trace.
```

Comment:
While this would be great syntax, google style docstrings don't make a distinction between args and kwargs. Could you instead put this as:
```
Args:
    name (str): Name of saved data trace to get.
    raw_data (bool, optional): option to return the h5_data directly 
        without interpreting it as a 2-D time trace.
```

Reply:
The pull request has been modified with these changes.